### ISSUE TITLE
Production & loss rates

### ISSUE THREAD

**Issue by SylvainPlessis** [libantioch/antioch#53]
The KineticsEvaluator object should be able to produce the production and loss terms of a species' rate.

This is a trivial thing to add, the only concern is technical, I need the Unit pull request (#24) to be merged first for the test regression value, the nice R value that changes to 10^-6 between the cal/mol value and this very same value when computed from J, so the test won't crash when merging it.

So basically, once the unit branch is merged, this will be a following pull request.
